### ISSUE TITLE
Armourer's Workshop conflicting with Custom NPC skins.

### ISSUE THREAD

**Issue by ghost** [RiskyKen/Armourers-Workshop#176]
Hey,

**TL;DR: Custom URL skins won't work with Custom NPC, but they will with More Player Models... Is there any way you can help?**

Sorry to bother but I want to use "Armourer's Workshop" and "Custom NPCs" together. Both of the mods seem to be working fine, but when I want to use a custom URL skin on an NPC it does not seem to load. I have the mod "More Player Models" (which has the same creator of Custom NPCs) and the custom URL skin shows to work normally. I'm not sure what the problem is, but I know that this mod is conflicting with the custom URLs.

The custom URL skins are 1024x1024 (because Custom NPCs and MPM both support higher resolution skins).

**Comment by PrimeEmerald** [comment]
Same here. But i have noticed they work together when you use a older version of the mod.

**Comment by RiskyKen** [comment]
try changing disableTexturePainting in the client config

**Comment by LordPhrozen** [comment]
Original user has deleted his account and wont see updates to this. @PrimeEmerald  can you test this?

**Comment by LordPhrozen** [comment]
No response from prime. original poster has no github account. closing.
